README.md

```diff
@@ -592,6 +592,24 @@ const PageSection = (props) => {
 };
 ```
 
+Additionally, you may encounter an issue with `SameSite` cookies not being set correctly. This can be solved by utilizing the code snippet below in your `~/pages/api/preview.js` file to replace `SameSite=Lax` to `SameSite=None; Secure;`.
```

Comment:
I would add information about the actual error - "`SameSite` cookies not being set correctly" is not easily findable (I haven't found it - though it is not part of the suggestion).

instead of `~/pages/api/preview.js` I would link Next.js [API route](https://nextjs.org/docs/api-routes/introduction)'s documentation, or even better - link [Preview Mode section](https://nextjs.org/docs/advanced-features/preview-mode) so that reader knows what the `~/pages/api/preview.js` route is - it is not reserved name for API route, it can be named by differently.

```suggestion
Additionally, you may encounter an issue with `SameSite` cookies not being set correctly. This can be solved by utilizing the code snippet below in your [API route handling preview](https://nextjs.org/docs/advanced-features/preview-mode) file to replace `SameSite=Lax` to `SameSite=None; Secure;` right after `res.setPreviewData` call.
```